lib/full_node.js

```diff
@@ -0,0 +1,14 @@
+//      
+var bcoin = require('bcoin');
+var TrustIsRisk = require('./trust_is_risk');
+
+class FullNode extends bcoin.fullnode {
+                     
+
+  constructor(options         ) {
```

Comment:
Don't check in compiled code. Instead, provide the script to compile it as part of your `package.json`.